rholang/src/main/scala/coop/rchain/rholang/interpreter/sort.scala

```diff
@@ -433,7 +433,7 @@ object MatchSortMatcher {
 object BundleSortMatcher {
   def sortMatch(b: Bundle): ScoredTerm[Bundle] = {
     val sortedPar = ParSortMatcher.sortMatch(b.body)
-    ScoredTerm(Bundle(sortedPar.term), Node(Score.BUNDLE, sortedPar.score))
+    ScoredTerm(b.copy(body = sortedPar.term), Node(Score.BUNDLE, sortedPar.score))
```

Comment:
You need to include the polarity in the sorting.

Reply:
What do you mean? Should the polarity influence the sorting or you mean I should preserve the polarity during sort"

Reply:
The polarity has to influence the sort order. Otherwise it won't normalize properly.